services/api/src/main/java/org/activiti/cloud/services/notifications/graphql/ws/api/GraphQLMessage.java

```diff
@@ -19,13 +19,26 @@
 import java.util.Map;
 import java.util.Objects;
 
+import javax.annotation.Generated;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonInclude.Include;
+
+@JsonInclude(value = Include.ALWAYS)
 public class GraphQLMessage {
 
 	private Map<String, Object> payload;
 	private String id;
 	private GraphQLMessageType type;
 
-	GraphQLMessage() {
+    @Generated("SparkTools")
```

Comment:
@igdianov what is this? are you using a library for that?

Reply:
@Salaboy I am using Spark Tools plugin in STS to generate builder code similar to built-in hashCode and equals generators

Reply:
`The Generated annotation is used to mark source code that has been generated.It can also be used to differentiate user written code from generated codein a single file. When used, the value element must have the name of thecode generator. The recommended convention is to use the fully qualifiedname of the code generator in the value field . `

Reply:
I have removed generated annotations.